streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -271,11 +274,16 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
   }
 
   /** Check if a sequence of numbers is in increasing order */
-  def isInIncreasingOrder(seq: Seq[Long]): Boolean = {
-    for (i <- 1 until seq.size) {
-      if (seq(i - 1) > seq(i)) {
+  def isInIncreasingOrder(data: Iterable[Long]): Boolean = {
```

Comment:
We could make it a one-liner (see http://stackoverflow.com/questions/7852471/idiomatic-construction-to-check-whether-a-collection-is-ordered ) but the one liners suggested don't short circuit on failure (although since we always expect this to be true the lack of short circuiting is probably ok).